auth-web/src/components/auth/staff/account-management/StaffPendingAccountsTable.vue

```diff
@@ -200,12 +201,16 @@ export default class StaffPendingAccountsTable extends Mixins(PaginationMixin) {
 
   private formatDate = CommonUtils.formatDisplayDate
 
-  @Watch('searchParams', { deep: true })
+  @Watch('searchParams', { deep: true, immediate: true })
```

Comment:
Moved this over to immediate, that way dealing with stuff inside mounted isn't necessary